core/model-loading/src/main/java/org/lgna/story/resourceutilities/JointedModelGltfExporter.java

```diff
@@ -73,6 +74,8 @@
 import org.lgna.project.io.JointedModelExporter;
 
 import javax.imageio.ImageIO;
+import java.awt.*;
```

Comment:
Narrow to just `import java.awt.Transparency;`
Importing all of awt makes List ambiguous with java.util.List.
Not sure why this is a problem when I build, but not for you.